### PR TITLE
Populate region_code and original_lab_address

### PR DESCRIPTION
**Pull request by islean**
## Description

When Mutant orders are placed, there is no way of setting region_code and original_lab_address when filling out samples automatically. There should be no need, though, as long as they provide a region and original_lab, which they are required to do. In the order form, as in this suggested solution, the fields are then automatically populated.

### Fixed

- If not provided, the region_code and original_lab_address are populated for the provided region and original_lab.

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b generate-mutant-fields -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/models/orders/samples.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 from cg.constants import DataDelivery
 from cg.constants.constants import GenomeVersion, Workflow
-from cg.models.orders.order import OrderType
+from cg.models.orders.constants import ORIGINAL_LAB_ADDRESSES, REGION_CODES, OrderType
```
Comment (islean) [inline cg/models/orders/samples.py:5]: Changed the import of OrderType as well since it lead to a circular import.

**Review by diitaz93 — APPROVED** [review]
Looks great 💯

**Comment by islean** [comment]
Tested in stage and all fields were set:

image

**Comment by islean** [comment]
Deployed to cg-services-prod.

**Pull request merged**